CONTRIBUTING.md

```diff
@@ -1,3 +1,119 @@
 # License
 
 By contributing your code, you agree to license your contribution under the MIT License.
+
+You also need to sign the [CLA](https://cla-assistant.io/bountysource/core) before your Pull Request is accepted.
+When you send your PR check the Pull Request page, it wil have a Link to signing the CLA.
+
+# How to Contribute
+Keep in mind this standardized setup is a work in progress, if you have any trouble please open an issue on the issue tracker and give as much info as possible.
+
+First of all you need [Docker](https://www.docker.com/what-docker) and [Docker Compose](https://docs.docker.com/compose/install/) installed.
+
+## Environment
```

Comment:
Not sure where it should go, I want to refactor it to a more appropriate place though. GitHub Wiki seems like a good idea.